pkg/provider/gardener/ruleset/disak8sstig/rules/242400.go

```diff
@@ -54,11 +54,11 @@ func (r *Rule242400) Name() string {
 func (r *Rule242400) Run(ctx context.Context) (rule.RuleResult, error) {
 	const option = "featureGates.AllAlpha"
 	var (
-		checkResults      []rule.CheckResult
-		deploymentNames   = []string{"kube-apiserver", "kube-controller-manager", "kube-scheduler"}
-		shootTarget       = rule.NewTarget("cluster", "shoot")
-		seedTarget        = rule.NewTarget("cluster", "seed")
-		kubeProxySelector = labels.SelectorFromSet(labels.Set{"role": "proxy"})
+		checkResults    []rule.CheckResult
+		deploymentNames = []string{"kube-apiserver", "kube-controller-manager", "kube-scheduler"}
+		shootTarget     = rule.NewTarget("cluster", "shoot")
+		seedTarget      = rule.NewTarget("cluster", "seed")
+		proxySelector   = labels.SelectorFromSet(labels.Set{"role": "proxy"})
```

Comment:
```suggestion
		kubeProxySelector = labels.SelectorFromSet(labels.Set{"role": "proxy"})
```
Please keep the old name for this selector as it is more specific.